tests/Integration/Behaviour/Features/Scenario/Cart/Calculation/RoundingType/round_line.feature

```diff
@@ -34,5 +34,5 @@ Feature: Cart calculation with rounding type LINE
     When I add 2 items of product "product2" in my cart
     When I add 3 items of product "product1" in my cart
     When I add 1 items of product "product3" in my cart
-    Then my cart total should be precisely 162.41 tax included
-    Then my cart total using previous calculation method should be precisely 162.41 tax included
+    Then my cart total should be precisely 162.4 tax included
```

Comment:
Maybe use `162.40` to avoid confusion (it would seem the decimals is rounded to one digit)